tests/pytests/test_profile.py

```diff
@@ -58,8 +58,7 @@ def testProfileSearch(env):
 
   # test PREFIX
   actual_res = conn.execute_command('ft.profile', 'idx', 'search', 'query', 'hel*', 'nocontent')
-  expected_res = ['Type', 'UNION', 'Query type', 'PREFIX - hel', 'Counter', 1, 'Child iterators', [
-                    ['Type', 'TEXT', 'Term', 'hello', 'Counter', 1, 'Size', 1]]]
+  expected_res = ['Type', 'TEXT', 'Term', 'hello', 'Counter', 1, 'Size', 1]
```

Comment:
Lovely as this is, I see we also loose important information here..
Maybe we should just change the print of these scenarios such that we get this information without the need for the `UNION` iterator wrapper.